spec/forms/sufia/forms/work_form_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe Sufia::Forms::WorkForm do
+describe Sufia::Forms::WorkForm, :no_clean do
```

Comment:
It's not *needed*, but it allows you to run this test without having fedora or solr running.